packages/storybook-utils/index.mjs

```diff
@@ -1 +1 @@
-export * from './dist/index.js';
+export * from './src/index.js';
```

Comment:
not sure how this worked before, but we use this package and exports work, so I assume that TS config changed at some point and stopped copying JS files into `dist`, however this is not needed really and we can just directly import from the `src`